crates/extensions/c8y_mapper_ext/src/config_operations.rs

```diff
@@ -171,22 +173,33 @@ impl CumulocityConverter {
                 };
 
                 let c8y_notification = Message::new(&smartrest_topic, smartrest_operation_status);
-                let clear_local_cmd = Message::new(&message.topic, "")
-                    .with_retain()
-                    .with_qos(QoS::AtLeastOnce);
-                vec![c8y_notification, clear_local_cmd]
+
+                if is_command_requester(cmd_id, REQUESTER_NAME) {
```

Comment:
Moment. If a command does not belong to the requester, the mapper should not even publish a status update message to c8y??? I think the mapper should just ignore the message.

Reply:
In 673135cac7ce86cecc4380699234141ead537675, I've also added checks to make sure Cumulocity operation transitions are not sent to Cumulocity for locally triggered operations.